### ISSUE TITLE
Bayesian Neural Networks Tensorflow 2.4.1 

### ISSUE THREAD

**Issue by luantunez** [tensorflow/probability#1305]
Hello and thank you for your work!
I was trying to implement a Bayesian Neural Network for classifying images with color depth. I am using Tensorflow 2.4.1 and Tensorflow Probability 0.12.1
No matter my architecture, learning rate, and loss function, my accuracy does not improve, is goes between 0.1 and 0.18, up and down. 
I think it is a problem with the gradients, that are not being calculated well for the probabilistic layers, like DenseFlipout(). Would you help me please?

Thank you, Lucia
